sql/migrations/warehouse/000037_add_idx_wh_staging_files_sid_did_upload_id_id_asc.up.sql

```diff
@@ -0,0 +1,3 @@
+-- Add index on wh_staging_files table to speed up queries on source_id, destination_id, upload_id, and id columns (e.g. for cron tracker query)
```

Comment:
`e.g. for cron tracker query)` -> There is another PR for cron tracker, rt? I think its okay to mention the exact query for which we are adding the index